charts/solr/templates/pvc-solr-configs.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: solr-configs
+spec:
+  accessModes:
+    - ReadWriteMany
+  storageClassName: azurefile
```

Comment:
storageClassName should note be tied to specific vendor.  Best to leave empty and allow it to use defaults.